commands/developer-utils/genpasswd.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Generate Password
+# @raycast.mode silent
+# @raycast.packageName Security
+
+# Optional parameters:
+# @raycast.icon images/genpasswd.png
+# @raycast.iconDark images/genpasswd-iconDark.png
+
+# Documentation:
+# @raycast.description Generates an iOS style password
+# @raycast.author everdrone
+# @raycast.authorURL https://github.com/everdrone
+
+import string
```

Comment:
To make use of argument as suggested
```suggestion
import sys
import string
```